api/http/dataplane/data-plane-api.yaml

```
@@ -145,26 +145,38 @@ components:
     ConfigurationStatus:
       type: object
       properties:
+        instanceId:
+          type: string
+          example: "aecea348-62c1-4e3d-b848-6d6cdeb1cb9c"
         correlationId:
           type: string
           example: "9e2d49c9-ada2-4ed1-b6f2-391d5cf634d9"
-        lastUpdated:
+        events:
+          type: array
+          items:
+            $ref: "#/components/schemas/Events"
+    Events:
```

This should be in the singular form, right?
```suggestion
    Event:
```